templates/default/reload.html

```diff
@@ -68,14 +68,16 @@ <H1><span class="bold">{$layout.pagetitle}</span></H1>
 				</TR>
 				{/foreach}
 				<TR class="dark ftop">
-					<TD colspan="2" class="nobr">
+					<TD colspan="5" class="nobr text-right">
 						<label>
-							<INPUT TYPE="checkbox" NAME="allbox" class="lms-ui-multi-check-all" VALUE="1">
 							{trans("Check All")}
+							<INPUT TYPE="checkbox" NAME="allbox" class="lms-ui-multi-check-all" VALUE="1">
 						</label>
 					</TD>
-					<TD colspan="3" class="buttons">
-						<A href="javascript:document.reload.submit();">{trans("Submit")} <IMG src="img/save.gif" ALT=""></A>
+				</TR>
+				<TR>
+					<TD class="text-right" colspan="5">
```

Comment:
Zmiana w niewłaściwym kierunku  - chcemy mieć klasę, która powoduje równanie do prawej panelu przycisków - i do tego było do tej pory buttons.

Reply:
Jak usunę tą klasę to przycisk leci do lewej. Jak powinno być poprawnie?

Reply:
Czy teraz będzie poprawnie?

Reply:
Przecież to klasa buttons powinna robić przy okazji text-right. Właśnie nie chcemy podawać wyrównań na sztywno tylko rozróżnić klasę wyglądu przycisków po to, żeby potem można było to całościowo zmienić tylko dla przycisków. 
Działający przykład masz tu:
https://github.com/chilek/lms-plus/blob/master/templates/default/rt/rtticketadd.html#L296
i jak widać równa panel przycisków do prawej. Nie wiem po co ta zmiana z buttons na text-right - ona moim zdaniem w ogóle nie jest potrzebna.